tests/test_fixtures/v3/notebook_assets/suite_edit/HEADER.md

```diff
@@ -0,0 +1,6 @@
+# Edit Your Expectation Suite
+Use this notebook to recreate and modify your expectation suite:
+
+**Expectation Suite Name**: `{{ suite_name }}`
+
+We'd love it if you **reach out to us on** the [**Great Expectations Slack Channel**](https://greatexpectations.io/slack)
```

Comment:
Lets remove this line to be consistent with the other commands

Reply:
Done (in next commit).